app/scripts/modules/core/src/presentation/Popover.tsx

```diff
@@ -3,10 +3,11 @@ import { OverlayTrigger, Popover as BSPopover } from 'react-bootstrap';
 
 import { Placement } from 'core/presentation/Placement';
 
-interface IProps {
+export interface IProps {
   value?: string;
   template?: JSX.Element;
   placement?: Placement;
+  container?: any;
```

Comment:
Probably.  It's defined as `any` in `@types/react-bootstrap`.

I didn't end up using `container` for alerts so I'm just going to nuke it.